### ISSUE TITLE
Should python.py make a call to 

### ISSUE THREAD

**Issue by pirapira** [raiden-network/raiden#4829]
When `api/python.py` calls the TokenNetworkRegistry proxy to [register a new token](https://github.com/raiden-network/raiden/blob/db54b197f8f225bc3936bc4c6052e1c4a041c28a/raiden/api/python.py#L230), the call does not specify a block identifier.  Should the call be made with a block identifier so that the precondition checks in `python.py` and in the proxy are synchronized?

**Comment by hackaugusto** [comment]
Yes, we should always specify the triggering block. Preconditions cannot be checked without this, because they would be susceptible to race conditions on many cases.
